keyvi/src/cpp/dictionary/fsa/generator.h

```diff
@@ -220,7 +218,7 @@ final {
         stack_->UpdateWeights(0, input_key.size() + 1, weight);
       }
 
-      last_key_ = key;
+      last_key_ = std::move(input_key);
```

Comment:
`input_key` is a `const`, so `std::move` call will return `const std::string&&` which is not the same as `std::string&&` and in the end anyway `copy-constructor` will be called as it's a better match, not `move`